scm/src/GFS_typedefs.F90

```diff
@@ -6968,6 +6984,17 @@ subroutine interstitial_phys_reset (Interstitial, Model)
       Interstitial%radar_reset = mod(Model%kdt-1, nint(Model%nsradar_reset/Model%dtp)) == 0
     end if
     !
+
+	!
+	! RRTMGP
+	!
+	if (Model%do_RRTMGP) then  
+	   if (Model%use_LW_jacobian) then 
+          !Interstitial%fluxlwUP_jac   = clear_val
```

Comment:
Why commented out? Same for line 6994.

Reply:
@dustinswales I am curious about this too.